crates/core/src/host/host_controller.rs

```diff
@@ -108,10 +118,19 @@ impl EnergyQuanta {
         self.0 as u64
     }
 
+    const WASMTIME_MULTIPLIER: u64 = 1_000;
+    pub fn as_wasmtime_points(&self) -> u64 {
+        self.as_wasmer_points() / Self::WASMTIME_MULTIPLIER
+    }
```

Comment:
Since `WASMTIME_MULTIPLIER` is used as both a product and a divisor, could you include the units in the name? I.e. `FOO_TO_WASMTIME_MULTIPLIER` or `WASMTIME_TO_FOO_MULTIPLIER` as appropriate, where `FOO` is... whatever `wasmer_points` gets changed to, I hope. Alternatively, add a docstring to the const which communicates that information.

Reply:
Before I saw this comment, was actually working on a different way of going about it that makes it more clear and moves it to `host::wasmtime`